libs/coin-modules/coin-aptos/src/api/index.ts

```diff
@@ -146,12 +147,22 @@ export class AptosAPI {
     return pendingTx.data.hash;
   }
 
-  private async getBalance(address: string): Promise<BigNumber> {
+  async getBalance(address: string, token: TokenCurrency): Promise<BigNumber> {
+    let balance = new BigNumber(0);
+    if (token.tokenType == "coin") {
```

Comment:
Normally I prefer to use strict equality
```suggestion
    if (token.tokenType === "coin") {
```